src/en/components/footer/code.md

```diff
@@ -31,22 +31,22 @@ Choose the **full display** if you need to include:
 ### Include the footer links band for Government of Canada sites and products
 
 - Always maintain the integrity of the signature. Absolutely avoid modifying the signature in any way. In particular avoid stretching or changing the colours or text.
-- Pass either the colour or white display mode of the Government of Canada signature using the signature-variant attribute. Choose the colour display for a white background.
-- Optionally, pass a boolean to set the Government of Canada signature link to Canada.ca using the signature-has-link attribute.
+- Pass either the `colour` or `white` display mode of the Government of Canada signature using the `signature-variant` attribute. Choose the `colour` display for a white background.
```

Comment:
I believe we can remove the second and third bullets from this section. The third one for sure can be removed since that applies to the header not the footer. 

Checking in figma to make sure.

Reply:
Just the third for now

Reply:
Removed the third bullet point only for now.